doc/array/pop.md

```diff
@@ -1,2 +1,24 @@
 @property {function} can-observe/pop pop
 @parent can-observe/array
+
+@description Remove an element from the end of an observe array.
+@signature `list.pop()`
+
+  `pop` removes an element from then end of an observe array.
```

Comment:
then -> the

It’d be nice to say it removes the _last_ element from the array.